contents/model-source

```diff
@@ -233,6 +233,9 @@ log.debug('filter: %s' % filter)
 
 ps_command=["docker","ps","--format",'"{{.ID}}"']
 
+if 'RD_CONFIG_ALL_CONTAINERS' in os.environ:
```

Comment:
maybe you need to check if the value is true or false because it could always be adding it

`if os.environ.get('RD_CONFIG_ALL_CONTAINERS') == 'true':`

Reply:
Done